src/explore-education-statistics-admin/src/pages/publication/PublicationManagePage.tsx

```diff
@@ -0,0 +1,120 @@
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import PublicationForm from '@admin/pages/publication/components/PublicationForm';
+import {
+  dashboardRoute,
+  legacyReleasesRoute,
+  PublicationRouteParams,
+  ThemeTopicParams,
+} from '@admin/routes/routes';
+import publicationService from '@admin/services/publicationService';
+import appendQuery from '@admin/utils/url/appendQuery';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import React from 'react';
+import { generatePath, RouteComponentProps, useHistory } from 'react-router';
+import RelatedInformation from '@common/components/RelatedInformation';
+
+const PublicationManagePage = ({
```

Comment:
Would keep this named as `PublicationEditPage` as nothing's really changed with this page and this diff gives the completely wrong impression that the entire thing is new.